docs/guides/backends.rst

```diff
@@ -0,0 +1,184 @@
+
+If you want to run multiple circuits with different number of samples, the code needs to be modified as follows:
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # QuantumBackend run and measure circuitset
+  :end-before: # End QuantumBackend run and measure circuitset
+
+``QuantumBackend`` can also provide the distribution of the outcome measurement. It can be extracted using the following approach:
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # Quantumbackend measurement distribution
+  :end-before: # End Quantumbackend measurement distribution
+
+In all instances, ``QuantumBackend`` automatically adds the measurement operators to the circuit. Therefore, the user is not required to include it in the circuit.
+
+Simulator-specific methods
+--------------------------
+
+``Quantumsimulator`` can be used to extract the wave function and expectation value of a given circuit. Below is an example of extracting these values using a ``Quantumsimulator`` called ``CirqSimulator``:
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # Quantumsimulator examples
+  :end-before: # End Quantumsimulator examples
+
+Zapata's backends
+=================
+
+SymbolicSimulator
+-----------------
+
+Zapata's ``SymbolicSimulator`` is a type of :ref:`quantum simulator <quantum_simulator>` that's built specifically to allow for the evaluation of circuits with :ref:`symbolic gates <symbolic_gates>`. Because there is extra overhead to allow for this broader type of evaluation, ``SymbolicSimulator`` is not the most performant simulator, but it can be useful for deeply understanding small circuits.
+
+The ``SymbolicSimulator`` can be used the same way any other simulator can be. To see more usage please refer to the :ref:`section in this guide on various methods <backend_methods>` or to the :doc:`API documentation </api/quantum/api/backend/index>`
+
+TrackingBackend
+---------------
+
+A ``TrackingBackend`` is a backend that tracks and stores data about each run of a circuit. This is accomplished by wrapping pre-existing backends in a ``TrackingBackend``. Currently the ``MeasurementTrackingBackend`` in :doc:`orquestra.quantum.trackers </api/quantum/trackers/index>` is the only implemented ``TrackingBackend``
+
+When ``run_circuit_and_measure`` is called, the tracking backend stores the ``data_type`` recieved from the call (usually a measurement outcome distribution), the wrapped device, the circuit which was run, the disribution of bitstrings which was recieved, the number of gates in the circuit, and the number of shots run in a JSON file. This is useful when you are paying to run circuits on a machine and you want to be able to re-use the data in a different workflow.
+
+To create a ``MeasurementTrackingBackend`` it must be initialized with a backend to be wrapped around, a name for the file you are storing the data in, and a boolean indicating whether or not the individual bitstrings should be saved. 
```

Comment:
```suggestion
``MeasurementTrackingBackend`` must be initialized with a backend to be wrapped around, a name for the file you are storing the data in, and a boolean indicating whether or not the individual bitstrings should be saved. 
```

Reply:
fixed in 4551c621e5c9c2b2295716c545eb6211b836df64 with slightly different language